src/components/Table.tsx

```diff
@@ -134,6 +192,15 @@ export default class Table extends React.Component<
 			<BaseTable {...props}>
 				<thead>
 					<tr>
+						{this.props.onChecked && (
+							<th>
+								<input
```

Comment:
I guess we could **in a later release** allow custom elements for this one as well.
Probably by making a union type for the column, where when `onChecked` is defined then
```ts
{
  ...
  render?: (...) =>` JSX.Element & { checked?: boolean; onChange: () => void };
  ...
}
```